pkger/parser_test.go

```diff
@@ -1455,6 +1832,206 @@ spec:
 				}
 			})
 		})
+
+		t.Run("pkg with single dashboard xy chart", func(t *testing.T) {
+			t.Run("xy chart", func(t *testing.T) {
```

Comment:
I'm a little confused by this big diff here. Thought these tests were all here already? Was thsi just reorded to be aplhabetical? If so, schaawweet 👌

Reply:
yes sir - re ordered things to make it a little easier on the eyes to follow!